### PR TITLE
NMS-13922: Added command to trigger SnmpMetadataProvisioningAdapter

### PR DESCRIPTION
**Pull request by christianpape**
* JIRA: https://opennms.atlassian.net/browse/NMS-13922

### CODE REVIEW

File: opennms-provision/opennms-provision-shell/src/main/java/org/opennms/netmgt/provision/requisition/command/SnmpMetadataProvisingAdapterCommand.java [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+import org.opennms.netmgt.provision.SnmpMetadataProvisioningAdapter;
+import org.springframework.transaction.TransactionStatus;
+import org.springframework.transaction.support.TransactionCallback;
+import org.springframework.transaction.support.TransactionOperations;
+
+import com.google.common.primitives.Booleans;
+
+@Command(scope = "opennms", name = "snmp-metadata-provisioning-adapter", description = "Trigger SnmpMetadataProvisioningAdapter for a set of nodes")
+@Service
+public class SnmpMetadataProvisingAdapterCommand implements Action {
```
Comment (synqotik) [inline opennms-provision/opennms-provision-shell/src/main/java/org/opennms/netmgt/provision/requisition/command/SnmpMetadataProvisingAdapterCommand.java:58]: Class and file name should be updated: `SnmpMetadataProvisingAdapterCommand` -> `SnmpMetadataProvisioningAdapterCommand`

**Review by synqotik — CHANGES_REQUESTED** [review]
Looks good, just needs class/file name fix.

File: integrations/opennms-snmp-metadata-provisioning-adapter/src/main/java/org/opennms/netmgt/provision/SnmpMetadataProvisioningAdapter.java [hunk 2]
```diff
@@ -102,6 +101,60 @@ public void doUpdateNode(final int nodeId) throws ProvisioningAdapterException {
         queryNode(nodeId);
     }
 
+    public List<OnmsMetaData> createMetadata(final OnmsNode node, final OnmsIpInterface primaryInterface) throws SnmpMetadataException {
```
Comment (synqotik) [inline integrations/opennms-snmp-metadata-provisioning-adapter/src/main/java/org/opennms/netmgt/provision/SnmpMetadataProvisioningAdapter.java:104]: Nice refactoring to method :)

File: opennms-provision/opennms-provision-shell/src/main/java/org/opennms/netmgt/provision/requisition/command/SnmpMetadataProvisingAdapterCommand.java (hunk 1 above)
Comment (christianpape) [reply opennms-provision/opennms-provision-shell/src/main/java/org/opennms/netmgt/provision/requisition/command/SnmpMetadataProvisingAdapterCommand.java:58]: OMG! Good catch!

Comment (christianpape) [reply opennms-provision/opennms-provision-shell/src/main/java/org/opennms/netmgt/provision/requisition/command/SnmpMetadataProvisingAdapterCommand.java]: done.

**Review by synqotik — APPROVED** [review]
LGTM!

**Pull request merged**